README.md

```diff
@@ -1 +1,17 @@
 # memprof
```

Comment:
Do you want to make this an Rmd README (usethis has a helper)

Reply:
Nice, thanks!